### PR TITLE
Update to latest version

### PR DESCRIPTION
**Pull request by mahesh-immanni**
Update latest version by syncing to upstream project

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,86 @@
+name: GitHub Actions CI
```
Comment (neil-greenwood) [inline .github/workflows/tests.yml:1]: I don't know if we need this file - I cannot see anything preventing random users DOS-ing by starting many actions.

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  schedule:
+    interval: daily
+  open-pull-requests-limit: 10
+  allow:
+  - dependency-type: direct
+  - dependency-type: indirect
+  ignore:
+  - dependency-name: omniauth-github
+    versions:
+    - "> 2.0.0, < 3"
```
Comment (neil-greenwood) [inline .github/dependabot.yml:14]: I wonder what this override is for...

**Review by neil-greenwood — APPROVED** [review]
In general, +1

File: .github/workflows/tests.yml (hunk 1 above)
Comment (mahesh-immanni) [reply .github/workflows/tests.yml:1]: This came from the upstream 
I am encountering issues running the tests as then are targetting macOS 11 which as per the below link is not open for new registrations
 [https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners]

File: .github/dependabot.yml (hunk 2 above)
Comment (mahesh-immanni) [reply .github/dependabot.yml:14]: This comes from the upstream

**Pull request merged**